react/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM node:20.10.0-alpine3.18 as node
+
+RUN mkdir /www
+COPY react /www
+WORKDIR /www
+
+# Creating dummy local file as it is not dynamically loaded; not actually used in prod  TODO_REACT
+RUN cp src/secret_local.example.ts src/secret_local.ts
```

Comment:
Shouldn't this be `COPY src/secret_local.example.ts src/secret_local.ts`

Reply:
Yes, makes sense 👍